native/cocos/editor-support/spine-wasm/spine-mesh-data.cpp

```diff
@@ -12,22 +12,19 @@ void SpineMeshData::initMeshMemory() {
     if (vBuf) return;
     const auto vCount = 65535;
     const auto byteStride = 7 * sizeof(float);
-    vBuf = new uint8_t[2 * vCount * byteStride];
-    iBuf = new uint16_t[8 * 65535];
+    vBuf = static_cast<uint8_t*>(::malloc(sizeof(uint8_t) * (2 * vCount * byteStride))); // 2 * 64KB * 28 = 3.5MB
+    iBuf = static_cast<uint16_t*>(::malloc(sizeof(uint16_t) * (8 * 65535))); // 512KB
```

Comment:
Use c malloc to allocate memory for raw data.

Reply:
65535 -> vCount